scripts/ci.sh

```diff
@@ -131,6 +131,8 @@ elif [ "$1" = "api-5" ]; then
 
     MOCHA_PARALLEL=true runJSTest "$1" $((2*$speedFactor)) $transcodingFiles $runnersFiles
 elif [ "$1" = "external-plugins" ]; then
+    pip install whisper-ctranslate2 openai-whisper jiwer
```

Comment:
This list of dependencies should be kept somewhere in the package directory instead.
(if it's not in the `requirements.txt` file at least in a script command)